scripts/run_unit_parametrized_tests.sh

```diff
@@ -1,10 +1,10 @@
 #! /bin/bash
 
-while getopts ":s:n:" opt; do
```

Comment:
Changes below looks very simmalar to scripts/run_integration_parametrized_tests.sh.
Can it be refactored to have as less as possible ammount of duplicated code?